### PR TITLE
Add xml comments to the template

### PR DESCRIPTION
**Pull request by akshita31**

### CODE REVIEW

File: Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs [hunk 1]
```diff
@@ -25,25 +29,26 @@ public class Program
...
+        internal static int DateTime()
         {
             #region DateTime
             Console.WriteLine(System.DateTime.Now);
             #endregion
             return 0;
         }
 
-        public static int EmptyRegion()
+        internal static int EmptyRegion()
```
Comment (colombod) [inline Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs]: InvalidRegion sounds like a better name than EmptyRegion

Comment (rchande) [reply Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs]: There's nothing invalid about an empty region?

File: Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs [hunk 2]
```diff
@@ -3,13 +3,17 @@
 
 namespace Microsoft.DotNet.Try.ProjectTemplate.Tutorial
 {
-    ///<param name="region">Takes in the --region option from the code fence options in markdown</param>
-    ///<param name="session">Takes in the --session option from the code fence options in markdown</param>
-    ///<param name="package">Takes in the --package option from the code fence options in markdown</param>
-    ///<param name="project">Takes in the --project option from the code fence options in markdown</param>
-    ///<param name="args">Takes in any additional arguments passed in the code fence options in markdown</param>
+    ///<summary>This program uses <a href="https://github.com/dotnet/command-line-api/wiki/DragonFruit-overview">System.CommandLine DragonFruit</a> to accept arguments from command line.
```
Comment (rchande) [inline Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs]: "accept command line arguments"

Comment (jonsequitur) [reply Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs]: `System.CommandLine DragonFruit` -> `System.CommandLine.DragonFruit`

File: Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs [hunk 3]
```diff
@@ -19,33 +23,26 @@ public class Program
         {
             return region switch
             {
-                "HelloWorld" => HelloWorld(),
-                "DateTime" => DateTime(),
-                _ => EmptyRegion()
-            };
+            "HelloWorld" => HelloWorld(),
```
Comment (loic-sharma) [inline Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs]: The formatting seems off here

Comment (akshita31) [reply Microsoft.DotNet.Try.ProjectTemplate/Tutorial/content/Program.cs]: Fixed. Thanks!

**Pull request merged**